src/main/java/carbon/error/CorruptedSavefileException.java

```diff
@@ -0,0 +1,26 @@
+package carbon.error;
+
+/**
+ * Exception class for corrupted stored data on tasks, that cannot be read.
+ */
+public class CorruptedSavefileException extends CarbonException {
+    private String data;
+
+    /**
+     * Constructs an instance of the exception.
+     *
+     * @param data String message for the error.
+     * @return CorruptedSavefileException object.
+     */
+    public CorruptedSavefileException(String data) {
+        super(data);
+        this.data = data;
+    }
+
+    /** @inheritDoc */
```

Comment:
I think it has to include curly braces
`{@inheritDoc}` but prof also said it's unnecessary in one of the forum posts if we don't want to write for overridden methods